jdaviz/conftest.py

```diff
@@ -156,6 +158,30 @@ def spectrum_collection(spectrum1d):
     return result
 
 
+@pytest.fixture
+def multi_order_spectrum_list(spectrum1d, spectral_orders=10):
+    sc = []
+
+    for i in range(spectral_orders):
+        np.random.seed(42)
+
+        spec_axis = (np.arange(SPECTRUM_SIZE) + 6000 + i * SPECTRUM_SIZE) * u.AA
+        flux = (np.random.randn(len(spec_axis.value)) +
+                10 * np.exp(-0.002 * (spec_axis.value - 6563) ** 2) +
+                spec_axis.value / 500) * u.Jy
+        uncertainty = StdDevUncertainty(np.abs(np.random.randn(len(spec_axis.value))) * u.Jy)
+        meta = dict(header=dict(FILENAME="jdaviz-test-multi-order-file.fits"))
+        spectrum1d = Spectrum1D(spectral_axis=spec_axis, flux=flux,
+                                uncertainty=uncertainty, meta=meta)
+
+        sc.append(spectrum1d)
+
+    with warnings.catch_warnings():
+        warnings.simplefilter('ignore')
```

Comment:
Why are we ignoring warnings here?

Reply:
I built the test on tutorials elsewhere in the docs where this was necessary. It's not anymore, so I've removed it. 👍🏻